source/includes/gcp/_health_checks.md

```diff
@@ -1,5 +1,145 @@
 ### Health checks
 
+<!-------------------- List HEALTH CHECK -------------------->
+#### List health checks
+
+```shell
+curl -X GET \
+  -H 'MC-Api-key: your_api_key'
+  "https://cloudmc_endpoint/v1/services/gcp/test-area/healthchecks"
+  # The above command returns JSON structured like this:
+```
+
+```json
+{
+  "data": [
+    {
+      "checkIntervalSec": 10,
+      "creationTimestamp": "2019-09-18T12:39:48.982-07:00",
+      "description": "",
+      "healthyThreshold": 2,
+      "id": "5930212998788364011",
+      "kind": "compute#healthCheck",
+      "name": "healthcheckname",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/global/healthChecks/healthcheckname",
+      "tcpHealthCheck": {
+        "port": 80,
+        "proxyHeader": "NONE",
+        "request": "",
+        "response": ""
+      },
+      "timeoutSec": 5,
+      "type": "TCP",
```

Comment:
As per description can only be HTTP or HTTPS

Reply:
no more TCP in the doc